notes_02-02-2016.md

```diff
@@ -16,3 +16,7 @@ The different parts should be based on our own insights and how the different pr
 He also said it was important to have peer review ensure that everyone is contributing to the project.
```

Comment:
I think what he mentioned is that we should have peer reviews on if someone is doing his work, rather than just the plain text. I told him that we just trust eachother to pull our own weight, and that we'll see how it works out.